spec/dummy/config/config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 8a97515bd4f99f3f1015539159e069c00cba636166257c367e42be26e7279650d1ce1954897ee86c8ac4991718565b984f2336b6511a2b1d37a524870e03bb56
```

Comment:
oh ok this is Rails thing I see, not related to implementation, no  problem then :)